cfme/common/vm_views.py

```diff
@@ -12,6 +12,65 @@
     CheckableManageIQTree, VersionPick, Version, CheckboxSelect, BaseEntitiesView)
 
 
+class NonEditableTable(Table):
+    """Wigdet for non-editable table. used for selecting value"""
+    def fill(self, values):
```

Comment:
@mfalesni is such implementation of fill/read fine? I've also left read_content to get table values - just in case.

Reply:
@mshriver please add you comment on this too if there're any